modules/understanding-platform-components.adoc

```diff
@@ -18,5 +18,5 @@ You can view the platform components definition in the {product-title-short} por
 The **Platform components configuration** section lists platform components in the following categories:
 
 * **Core system components**: These components are part of the core {ocp} and Kubernetes namespaces. {product-title-short} includes them in the platform definition by default. You cannot customize these definitions. These definitions might change when you upgrade the system.
-* **Red{nbsp}Hat layered products**: Components found in Red{nbsp}Hat layered and partner product namespaces are included in the platform definition by default. You must update the definition if you install Red{nbsp}Hat products, including {product-title-short}, into a non-default namespace.
+* **Red{nbsp}Hat layered products**: Components found in Red{nbsp}Hat layered and partner product namespaces are included in the platform definition by default. Red{nbsp}Hat recommends that you update the definition if you install Red{nbsp}Hat products, including {product-title-short}, into a non-default namespace.
```

Comment:
Both the IBM and RH SSG have gripes around the use of the word "Recommends": 

![Screenshot From 2025-07-09 12-41-21](https://github.com/user-attachments/assets/95380288-efa2-47db-b8fc-61bc6b39124e)

How about:

> Ensure that you update the definition if...